.travis.yml

```diff
@@ -171,14 +171,127 @@ matrix:
       python: 3.7
       stage: integration
       env: 
-          - TOXENV=behave-azure-pro
+          - TOXENV=behave-azurepro-14.04
           - UACLIENT_BEHAVE_DEBS_PATH=deb-artifacts
       workspaces:
         use:
           - trusty
+      install:
+          - make testdeps
+          - git clone https://github.com/canonical/server-test-scripts.git
+          - git clone https://github.com/canonical/pycloudlib.git
+          - cd pycloudlib
+          - pip install -r requirements.txt
+          - pip install -e .
+          - cd ..
+      script:
+          - mkdir deb-artifacts
+          - cp *-debs/* deb-artifacts
+          - >
+              if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+                  BUILD_PR=0
+              else
+                 cd $TRAVIS_BUILD_DIR/..
+                 tar -zcf pr_source.tar.gz ubuntu-advantage-client
+                 cp pr_source.tar.gz /tmp
+                 ls -lh /tmp
+                 cd $TRAVIS_BUILD_DIR
+              fi
+          - ls deb-artifacts
+          - sudo apt-get remove --yes --purge lxd lxd-client
+          - sudo rm -Rf /var/lib/lxd
+          - UACLIENT_BEHAVE_BUILD_PR=${BUILD_PR} make test
+          - echo $TRAVIS_JOB_NUMBER
+          - python3 server-test-scripts/ubuntu-advantage-client/azure_cleanup.py --suffix-tag ${TRAVIS_JOB_NUMBER/./-} --client-id ${UACLIENT_BEHAVE_AZ_CLIENT_ID} --client-secret ${UACLIENT_BEHAVE_AZ_CLIENT_SECRET} --subscription-id ${UACLIENT_BEHAVE_AZ_SUBSCRIPTION_ID} --tenant-id ${UACLIENT_BEHAVE_AZ_TENANT_ID} || true
+    - if: env(UACLIENT_BEHAVE_AZ_CLIENT_SECRET)
```

Comment:
Let's move this down into it's own allow_failure section like this:
https://paste.ubuntu.com/p/jg4YNMddqB/